Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -201,17 +206,3 @@ extension PopupView {
         }
     }
 }
-
-extension PopupView {
-    /// Specifies whether a "close" button should be shown to the right of the popup title. If the "close"
-    /// button is shown, you should pass in the `isPresented` argument to the `PopupView`
-    /// initializer, so that the the "close" button can close the view.
-    /// Defaults to `false`.
-    /// - Parameter newShowCloseButton: The new value.
-    /// - Returns: A new `PopupView`.
-    public func showCloseButton(_ newShowCloseButton: Bool) -> Self {
-        var copy = self
-        copy.showCloseButton = newShowCloseButton
-        return copy
-    }
-}
```

Comment:
Moved to `PopupView+ShowCloseButton.swift` to be consistent with how other PopupView and FeatureFormView modifiers are written.